MrtCore/console_unpackaged_app/console_unpackaged_app.cpp

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include <windows.h>
+#include <iostream>
+
+#include "winrt\Windows.Foundation.h"
+#include "winrt\Windows.Foundation.Collections.h"
+#include "winrt\Microsoft.ApplicationModel.Resources.h"
+
+#include <MddBootstrap.h>
+
+using namespace winrt;
+using namespace winrt::Microsoft::ApplicationModel::Resources;
+
+HRESULT MddBootstrapInitializeWrapper()
+{
+    // Take a dependency on Project Reunion v0.8 preview.
+    const UINT32 majorMinorVersion{ 0x00000008 };
+    PCWSTR versionTag{ L"preview" };
+    const PACKAGE_VERSION minVersion{};
+    HRESULT hr{ MddBootstrapInitialize(majorMinorVersion, versionTag, minVersion) };
+    if (FAILED(hr))
+    {
+        wprintf(L"Error 0x%X in MddBootstrapInitialize(0x%08X, %s, %hu.%hu.%hu.%hu)\n",
+            hr, majorMinorVersion, versionTag, minVersion.Major, minVersion.Minor, minVersion.Build, minVersion.Revision);
+        return hr;
+    }
+
+    return S_OK;
+}
+
+ResourceManager InitMrtCore()
+{
+    // Required for C++/WinRT. This call associates this thread with an apartment and initializes COM runtime.
+    init_apartment();
+
+    // Create a resource manager using the resources index (PRI file) generated during build.
+    auto factory = winrt::get_activation_factory<ResourceManager, IResourceManagerFactory>();
+    return factory.CreateInstance(L"console_unpackaged_app.pri");
+}
+
+int wmain(int argc, wchar_t* argv[])
+{
+    // Print usage help.
+    if ((argc < 2) || (wcscmp(argv[1], L"--help") == 0) || (wcscmp(argv[1], L"-?") == 0))

```

Comment:
Another interesting mode might be "user passes in a resource name, this app prints out the found resource and its string" .. so you'd write `console_unpackaged_app lookup Resources/SampleString` and it would print the "sample string" value.

Reply:
Will try to return to this app soon in a separate change and add this mode.